Evaluating_pipeline.md

```diff
@@ -0,0 +1,191 @@
+# Evaluation of the variant remapping pipeline
+
+To assess the quality of the remapping pipeline we remap GiaB variants from GRCh37 to GRCh38 
+and compare them with GiaB variants from GRCh38. To ensure that the remapped and standard variants are comparable 
+We first filter the gold standard to keep only variants that are present in both dataset.
```

Comment:
I think the EVA standard for Markdown documentation is that we don't wrap the lines to any fixed width, but instead rely on soft wrapping in IDE & on GitHub. This makes editing the files easier as the changes don't necessitate rewrapping the entire paragraph, and also word diffs are clearer to read.

```suggestion
To assess the quality of the remapping pipeline we remap GiaB variants from GRCh37 to GRCh38 and compare them with GiaB variants from GRCh38. To ensure that the remapped and standard variants are comparable, we first filter the gold standard to keep only variants that are present in both datasets.
```